main.tf

```diff
@@ -239,7 +241,7 @@ resource "local_file" "kubeconfig" {
 # Module Registry - https://registry.terraform.io/modules/GoogleCloudPlatform/sql-db/google/12.0.0/submodules/postgresql
 module "postgresql" {
   source     = "GoogleCloudPlatform/sql-db/google//modules/postgresql"
-  version    = "~> 18.2.0"
+  version    = "~> 19.0.0"
```

Comment:
Is this a potential breaking change as well?

Reply:
The v19.0.0 release is unique in that it is breaking for MySQL module, but postgres is fine. The update here does not cause any issues, see
https://github.com/terraform-google-modules/terraform-google-sql-db/blob/v19.0.0/CHANGELOG.md#1900-2024-02-08